bot/university/university.py

```diff
@@ -248,9 +251,9 @@ async def list_exchanges(self, ctx: commands.Context):
         Args:
             ctx (discord.ext.commands.Context): The context from which this command is invoked.
         """
-        requests_of_user = self._db_connector.get_group_exchange_for_user(ctx.author.id)
-        if requests_of_user:
-            embed = await self._build_group_exchange_list_embed(requests_of_user)
+        exchange_requests = self._db_connector.get_group_exchange_for_user(ctx.author.id)
```

Comment:
Rename to `user_exchange_requests`

Reply:
Why? I think because of the method call it's clear whose exchange requests they are.